service/resource/deployment/metric.go

```diff
@@ -0,0 +1,29 @@
+package deployment
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	PrometheusNamespace       = "kvm_operator"
+	PrometheusSubsystem       = "deployment_resource"
+	VersionBundleVersionLabel = "giantswarm.io/version-bundle-version"
```

Comment:
I don't get why this constant is here. spec.go or some such would make more sense to me

Reply:
I would like to keep this here since the version label is used in different places and associated to the metrics magic. Spec would not make sense to me somehow.